GVFS/GVFS/CommandLine/LogVerb.cs

```diff
@@ -113,7 +113,7 @@ private void DisplayMostRecent(string logFolder, string logFileType)
         {
             string logFile = FindNewestFileInFolder(logFolder, logFileType);
             this.Output.WriteLine(
-                "  {0, -10}: {1}",
+                "  {0, -15}: {1}",

```

Comment:
It would be nice if we could find a way not to hard code this and use the known `logFileTypes`.  If not will you add a comment about this number, why it is `-15` so we will know how we came up with it in case we need to change it again.

Reply:
Replaced the hardcoding with dynamic computation of width using `logFileTypes`.